custom/icds_reports/models/aggregate.py

```diff
@@ -1520,3 +1522,33 @@ class Meta(object):
         db_table = AGG_DASHBOARD_ACTIVITY
 
     _agg_helper_cls = DashboardActivityReportAggregate
+
+
+class AggregateMigrationForms(models.Model, AggregateMixin):
+    """Aggregated data for migration
+
+    A migration table exists for each state_id and month.
+
+    A row exists for every child_health case that has had a THR Form
+    submitted against it this month.
+    """
+
+    # partitioned based on these fields
+    state_id = models.CharField(max_length=40)
+    supervisor_id = models.TextField(null=True)
+    month = models.DateField(help_text="Will always be YYYY-MM-01")
+
+    # not the real pkey - see unique_together
+    case_id = models.CharField(max_length=40, primary_key=True)
```

Comment:
Will be clearer to say it as person_case_id as its coming from migration form and person case id is pulled from there so that its not confused with any other case id